db/migrate/20230320213156_create_crypto_address_for_channels.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCryptoAddressForChannels < ActiveRecord::Migration[7.0]
+  def change
+    create_table :crypto_address_for_channels, id: :uuid, default: -> { "uuid_generate_v4()"} do |t|
+      t.string :chain, null: false
```

Comment:
We should consider an enum here https://github.com/bibendi/activerecord-postgres_enum

to enforce data integrity outside of the app

Reply:
that's a good idea, I'll make some adjustments.